wazo_dird_client/commands/directories.py

```diff
@@ -31,11 +35,15 @@ def lookup_user(self, profile, user_uuid, token=None, tenant_uuid=None, **kwargs
 
         return r.json()
 
-    def reverse(self, profile, xivo_user_uuid, token=None, tenant_uuid=None, **kwargs):
-        url = '{base_url}/reverse/{profile}/{xivo_user_uuid}'.format(
+    def reverse(self, profile, user_uuid=None, token=None, tenant_uuid=None, **kwargs):
+        if not user_uuid and 'xivo_user_uuid' in kwargs:
+            logger.warning('The "xivo_user_uuid" argument has been renamed to "user_uuid"')
+            user_uuid = user_uuid or kwargs.pop('xivo_user_uuid')
```

Comment:
We already doing the check if user_uuid is None
```suggestion
            user_uuid = kwargs.pop('xivo_user_uuid')
```

Reply:
I applied your suggestion and force pushed with no other change